config/crd/common/kustomization.yaml

```diff
@@ -1,7 +1,6 @@
-# Code generated in runtime. DO NOT EDIT.
+# This file is NOT auto-generated
```

Comment:
This file has been reverted? [This](https://github.com/aws-controllers-k8s/runtime/blob/main/config/crd/kustomization.yaml) is what it should be

Reply:
I simply ran `make build-controller SERVICE=rds` from the code-generator repo...

Reply:
Your runtime repository must have been very out of date. It copies from that directory